CHANGELOG.md

```diff
@@ -1 +1 @@
-See [CHANGELOG/README.md](./CHANGELOG/README.md).
\ No newline at end of file
+See the [1.10 changelog in the master branch](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/CHANGELOG/CHANGELOG-1.10.md).
```

Comment:
This should be 1.0.md

https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/CHANGELOG/CHANGELOG-1.0.md